card/db/migrate_core_cards/20180508210903_add_json_type.rb

```diff
@@ -0,0 +1,10 @@
+# -*- encoding : utf-8 -*-
+
+class AddJsonType < Card::Migration::Core
+  def up
+    ensure_card "json", codename: "json", type_id: Card::CardtypeID
```

Comment:
some potential for conflict.  we probably need a "*" policy...

Reply:
(meaning some sites might have a "json" card)

Reply:
we don't use "*" in types. It doesn't look so pretty and maybe confusing in the type select field if there is one type with a "*"

Reply:
Yeah, this is definitely consistent with what we're doing now and ready to merge. I'm just wondering if we rethink how we use `*`s so we reduce name conflict.  (we could make the `*` pretty).